cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -274,6 +274,10 @@ data:
   # Whether the OBC provisioner should watch on the operator namespace or not, if not the namespace of the cluster will be used
   ROOK_OBC_WATCH_OPERATOR_NAMESPACE: "true"
 
+  # Enable volume replication controller
+  CSI_ENABLE_VOL_REPLICATION: "false"
```

Comment:
We should set this to true in some integration tests. Let me get back to you on the specifics after #7390 is merged.

Reply:
ack

Reply:
We can only enable it on openshift, right? The setting can be replaced in the operator.yaml, see the method `replaceOperatorSettings()` where you could call 
```go
if utils.IsPlatformOpenShift() {
  // replace the setting in the yaml
  manifest = strings.ReplaceAll(...)
}
```

Reply:
This can be enabled on kubernetes and OCP. this is not OCP specific